server/statistics/hot_peer_cache.go

```diff
@@ -69,16 +78,25 @@ type hotPeerCache struct {
 	topNTTL            time.Duration
 	reportIntervalSecs int
 	taskQueue          chan FlowItemTask
+	defaultThresholds  []float64
+	thresholdsOfStore  map[uint64]*thresholdWithTime // storeID -> thresholds
 }
 
 // NewHotPeerCache creates a hotPeerCache
 func NewHotPeerCache(kind RWType) *hotPeerCache {
+	statKinds := kind.RegionStats()
+	defaultThresholds := make([]float64, DimLen)
+	for dim, kind := range statKinds {
+		defaultThresholds[dim] = MinHotThresholds[kind]
+	}
 	c := &hotPeerCache{
-		kind:           kind,
-		peersOfStore:   make(map[uint64]*TopN),
-		storesOfRegion: make(map[uint64]map[uint64]struct{}),
-		regionsOfStore: make(map[uint64]map[uint64]struct{}),
-		taskQueue:      make(chan FlowItemTask, queueCap),
+		kind:              kind,
+		peersOfStore:      make(map[uint64]*TopN),
+		storesOfRegion:    make(map[uint64]map[uint64]struct{}),
+		regionsOfStore:    make(map[uint64]map[uint64]struct{}),
+		taskQueue:         make(chan FlowItemTask, queueCap),
+		defaultThresholds: defaultThresholds,
```

Comment:
I think the default thresholds can remove, how about put this values into the thresholds